packages/examples/cvat/exchange-oracle/src/services/cloud/gcs.py

```diff
@@ -10,15 +6,18 @@
 
 from src.services.cloud.client import StorageClient
 
+DEFAULT_GCS_HOST = "storage.googleapis.com"
+
 
-class GCSClient(StorageClient):
+class GcsClient(StorageClient):
```

Comment:
I agree that this is acronym and in a standalone situation the use of uppercase GCS would be natural. But when it is in a camel-case variable name with several words, I find it's more readable to start each word from a capital letter and then follow it with lowercase, even for acronyms. It becomes even more sensible if you have several acronyms in 1 name, e.g. `HTTPURLEncoder` vs. `HttpUrlEncoder`. When the acronym is in the middle of the name, it's also more readable to follow common camel case: `BasicHTTPEncoder` vs. `BasicHttpEncoder`.

Speaking about PEP, none of our projects actually relies on or explicitly derives from this PEP style guide, though it may have some good ideas to borrow.